cluster-provision/gocli/cmd/run.go

```diff
@@ -83,6 +87,7 @@ func NewRunCommand() *cobra.Command {
 	run.Flags().String("container-org", "kubevirtci", "the organization at the registry to pull the container from")
 	run.Flags().String("container-suffix", "", "Override container suffix stored at the cli binary")
 	run.Flags().String("gpu", "", "pci address of a GPU to assign to a node")
+	run.Flags().Bool("run-etcd-on-memory", true, "configure etcd to run on RAM memory, etcd data will not be persistent")
```

Comment:
Can you make this an it to specify MBs? The boolean is nice, but I saw in #498 that we may have to vary the size. Maybe just add another flag for that.

Reply:
No problem